source/fundamentals/crud/write-operations/insert.txt

```diff
@@ -4,9 +4,188 @@ Insert a Document
 
 .. default-domain:: mongodb
 
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+   
 Overview
 --------
 
-In this section, we show you how to call the write operations to **insert**
-documents from a collection in your MongoDB database.
+Before you can retrieve, update, or delete information in MongoDB, you
+first need to perform an **insert operation** to load data. An insert
```

Comment:
Issue: This sentence seems a bit long and a bit confusing without the context on the other drivers pages.
I can see a user wondering "What kind of information/data?"

Reply:
I added the same examples of information that are present on the Node pg but it sounds a bit odd now. Wouldn't you say that the kind of information would be specific to each user so they wouldn't wonder about what kind? Maybe another solution is to say documents instead of information.

Reply:
Agreed, I think changing it to say "document" solves this! I'll show what I have in mind in the next review